content/sensu-enterprise-dashboard/2.9/rbac/overview.md

```diff
@@ -226,6 +227,20 @@ type           | String
 allowed values | any length string that only contains URL-friendly characters. _PRO TIP: we recommend using a random string generator for access tokens; e.g.: <br><code>openssl rand -base64 40 &#124;  tr -- '+=/' '-&#95;~'</code>._
 example        | {{< highlight shell >}}"accessToken": "OrIXC7ezuq0AZKoRHhf~oIl-98dX5B23hf8KudfcqJt5eTeQjDDGDQ__"{{< /highlight >}}
 
+fallback    | 
+---------------|------
+description    | Used to give an authenticated user the attributes defined in that role if that user is not found in any other defined dashboard role.
+required       | false
+type           | Boolean
+default        | false
```

Comment:
```suggestion
default        | `false`
```

Reply:
Other default entries for Boolean's don't have the those around them. Want them on the rest in this guide? 

Also, this whole line might not even be needed looking at the PR for the feature:
https://github.com/sensu/sensu-enterprise-dashboard/pull/53

@palourde can you confirm if the fallback attribute even exists without actually being defined in a role? Thanks!